hypervisor/include/arch/x86/mmu.h

```diff
@@ -10,214 +10,68 @@
-#define     IA32E_EPT_SUPPRESS_VE           0x8000000000000000UL
-
-/* Definitions common or ignored for all IA-32e related paging entries */
-#define     IA32E_COMM_D_BIT                0x0000000000000040UL
-#define     IA32E_COMM_G_BIT                0x0000000000000100UL
-
-/* Definitions exclusive to a Page Map Level 4 Entry (PML4E) */
-#define     IA32E_PML4E_INDEX_MASK_START    39
-#define     IA32E_PML4E_ADDR_MASK           0x0000FF8000000000UL
-
 /* Definitions exclusive to a Page Directory Pointer Table Entry (PDPTE) */
 #define     IA32E_PDPTE_D_BIT               0x0000000000000040UL
 #define     IA32E_PDPTE_PS_BIT              0x0000000000000080UL
 #define     IA32E_PDPTE_PAT_BIT             0x0000000000001000UL
 #define     IA32E_PDPTE_ADDR_MASK           0x0000FFFFC0000000UL
-#define     IA32E_PDPTE_INDEX_MASK_START  \
-		(IA32E_PML4E_INDEX_MASK_START - IA32E_INDEX_MASK_BITS)
 
 /* Definitions exclusive to a Page Directory Entry (PDE) 1G or 2M */
 #define     IA32E_PDE_D_BIT                 0x0000000000000040UL
 #define     IA32E_PDE_PS_BIT                0x0000000000000080UL
 #define     IA32E_PDE_PAT_BIT               0x0000000000001000UL
 #define     IA32E_PDE_ADDR_MASK             0x0000FFFFFFE00000UL
-#define     IA32E_PDE_INDEX_MASK_START    \
-		(IA32E_PDPTE_INDEX_MASK_START - IA32E_INDEX_MASK_BITS)
 
 /* Definitions exclusive to Page Table Entries (PTE) */
 #define     IA32E_PTE_D_BIT                 0x0000000000000040UL
 #define     IA32E_PTE_PAT_BIT               0x0000000000000080UL
 #define     IA32E_PTE_G_BIT                 0x0000000000000100UL
 #define     IA32E_PTE_ADDR_MASK             0x0000FFFFFFFFF000UL
-#define     IA32E_PTE_INDEX_MASK_START    \
-		(IA32E_PDE_INDEX_MASK_START - IA32E_INDEX_MASK_BITS)
-
-/** The 'Present' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_P       0x00000001U
-/** The 'Read/Write' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_RW      0x00000002U
-/** The 'User/Supervisor' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_US      0x00000004U
-/** The 'Page Write Through' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_PWT     0x00000008U
-/** The 'Page Cache Disable' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_PCD     0x00000010U
-/** The 'Accessed' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_A       0x00000020U
-/** The 'Dirty' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_D       0x00000040U
-/** The 'Page Size' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_PS      0x00000080U
-/** The 'Global' bit in a 32 bit paging page directory entry */
-#define MMU_32BIT_PDE_G       0x00000100U
-/** The 'PAT' bit in a page 32 bit paging directory entry */
-#define MMU_32BIT_PDE_PAT     0x00001000U
+
 /** The flag that indicates that the page fault was caused by a non present
  * page.
```

Comment:
This comments seems to be confusing

Reply:
I think it wants to say "was caused by the present bit not set". 
This patch doesn't plan to resolve all the issues in mmu.h. Let's by pass this now.